backends/dpdk/dpdkAsmOpt.cpp

```diff
@@ -163,4 +163,35 @@ const IR::IndexedVector<IR::DpdkAsmStatement> *RemoveLabelAfterLabel::removeLabe
     }
     return new_l;
 }
+
+const IR::Node* RemoveUnusedMetadataFields::preorder(IR::DpdkAsmProgram *p) {
+    IR::IndexedVector<IR::DpdkStructType> usedStruct;
+    bool isMetadataStruct = false;
```

Comment:
I would call this "foundMetadataStruct".
Perhaps you can change it in a future PR.